trulens_eval/trulens_eval/app.py

```diff
@@ -665,6 +665,36 @@ async def main_acall(self, human: str) -> str:
 
         raise NotImplementedError()
 
+    def _extract_content(self, value):
```

Comment:
Can you add a note of the libs/apps that produce the types of objects that you are checking for here?

Reply:
Will add a note. Also updated the PR description with this information.